src/pkg/packaging/dir.props

```diff
@@ -9,26 +9,27 @@
     <SharedHostPublishRoot>$(IntermediateOutputRootPath)sharedHost/</SharedHostPublishRoot>
     <HostFxrPublishRoot>$(IntermediateOutputRootPath)hostFxr/</HostFxrPublishRoot>
     <SharedFrameworkPublishRoot>$(IntermediateOutputRootPath)sharedFx/</SharedFrameworkPublishRoot>
-    <CombinedPublishRoot>$(IntermediateOutputRootPath)combined-framework-host/</CombinedPublishRoot>   
+    <CombinedPublishRoot>$(IntermediateOutputRootPath)combined-framework-host/</CombinedPublishRoot>
   </PropertyGroup>
 
   <PropertyGroup>
     <ProductMoniker>$(PackageTargetRid)-$(HostFullVersion)</ProductMoniker>
+    <PortableMoniker Condition="'$(PortableBuild)' == 'true'">-portable</PortableMoniker>
```

Comment:
Sorry, I should correct myself. All the builds that used to have conflicting assets are now only portable. So we don't need to differentiate the assets anymore.

You are right, we are building Ubuntu and Debian because they produce the Deb installers for those platforms.  But that will soon change. Rakesh is working on creating the Deb installers after the portable Linux build.